s/elements/bindings-editor/views/buttons.ts

```diff
@@ -10,6 +10,7 @@ export const ButtonsView = view(use => (keybinds: Bindings, setKeybinds: StateSe
 	const applyBindings = () => {
 		localStorage.setItem('bindings', JSON.stringify(keybinds));
 		setKeybinds(loadBindings())
+		window.dispatchEvent(new Event("storage"));
```

Comment:
yeah so i needed to dispatch that event because the storage listener wasnt running when i added to storage, ive read that it doesnt run if these things happens in the same context but they werent ?

Reply:
let's remove the dispatching of storage events
  - keep listening to storage events, that's a good idea, no problems there (we just need to stop dispatching storage events)
  - instead of dispatching an event just to trigger our own handler, let's just call that handler functionality directly. we don't need to use the event system to "talk to ourselves" so to speak.